packages/factom-did-provider/src/__tests__/factom-did-provider.test.ts

```diff
@@ -0,0 +1,119 @@
+import { createAgent, IDIDManager, IIdentifier, IKeyManager, IKeyManagerCreateArgs, IResolver, ManagedKeyInfo, TAgent } from '@veramo/core'
+import { DIDManager, MemoryDIDStore } from '@veramo/did-manager'
+import { FactomDIDProvider, IRequiredContext } from '../factom-did-provider'
+import { getUniResolver } from '@sphereon/did-uni-client'
+import { KeyManager, MemoryKeyStore, MemoryPrivateKeyStore } from '@veramo/key-manager'
+import { KeyManagementSystem } from '@veramo/kms-local'
+import { DIDResolverPlugin } from '@veramo/did-resolver'
+import { Resolver } from 'did-resolver'
+
+const PUBLIC_KEY_HEX = '89a4661e446b46401325a38d3b20582d1dd277eb448a3181012a671b7ae15837'
+const LTO_DID = 'did:factom:5e2dd72ba81bb9cc50d500e148fdbcc44b09807336d3ebb9ea12451987cc4c00'
```

Comment:
think this should be called FACTOM_DID? and FACTOM_KID below?

Reply:
Yes.
Will clean that up after this PR as well, just like the documentation. It got pulled into this PR since the tests depend on the Factom and LTO code, but the actual Factom implementation is not finished yet, so stuff like this will be resolved once we have the full support in the SDK

Reply:
Removed the constants altogether